<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_s">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>871</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: rgb(176, 18, 235);
font: 14pt &quot;Magnum&quot;;
border-radius: 15px;
border: 2px solid rgb(176, 18, 235);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Сейчас играет: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_c">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(176, 18, 235);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
         <item>
          <widget class="QFrame" name="frame">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="Line" name="line_1">
            <property name="geometry">
             <rect>
              <x>39</x>
              <y>10</y>
              <width>25</width>
              <height>279</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(176, 18, 235, 64);
background-color: rgb(176, 18, 235);
font: 14pt &quot;Magnum&quot;;
border-radius: 15px;
border: 2px solid rgb(176, 18, 235);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>25</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
           <widget class="Line" name="line_5">
            <property name="geometry">
             <rect>
              <x>227</x>
              <y>10</y>
              <width>25</width>
              <height>279</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(176, 18, 235, 64);
background-color: rgb(176, 18, 235);
font: 14pt &quot;Magnum&quot;;
border-radius: 15px;
border: 2px solid rgb(176, 18, 235);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
           <widget class="Line" name="line_2">
            <property name="geometry">
             <rect>
              <x>86</x>
              <y>10</y>
              <width>25</width>
              <height>279</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(176, 18, 235, 64);
background-color: rgb(176, 18, 235);
font: 14pt &quot;Magnum&quot;;
border-radius: 15px;
border: 2px solid rgb(176, 18, 235);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>25</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
           <widget class="Line" name="line_4">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>10</y>
              <width>25</width>
              <height>279</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(176, 18, 235, 64);
background-color: rgb(176, 18, 235);
font: 14pt &quot;Magnum&quot;;
border-radius: 15px;
border: 2px solid rgb(176, 18, 235);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>25</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
           <widget class="Line" name="line_8">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>368</x>
              <y>10</y>
              <width>25</width>
              <height>279</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(176, 18, 235, 64);
background-color: rgb(176, 18, 235);
font: 14pt &quot;Magnum&quot;;
border-radius: 15px;
border: 2px solid rgb(176, 18, 235);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
           <widget class="Line" name="line_7">
            <property name="geometry">
             <rect>
              <x>321</x>
              <y>10</y>
              <width>25</width>
              <height>279</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(176, 18, 235, 64);
background-color: rgb(176, 18, 235);
font: 14pt &quot;Magnum&quot;;
border-radius: 15px;
border: 2px solid rgb(176, 18, 235);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
           <widget class="Line" name="line_6">
            <property name="geometry">
             <rect>
              <x>274</x>
              <y>10</y>
              <width>25</width>
              <height>279</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(176, 18, 235, 64);
background-color: rgb(176, 18, 235);
font: 14pt &quot;Magnum&quot;;
border-radius: 15px;
border: 2px solid rgb(176, 18, 235);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
           <widget class="Line" name="line_3">
            <property name="geometry">
             <rect>
              <x>133</x>
              <y>10</y>
              <width>25</width>
              <height>279</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(176, 18, 235, 64);
background-color: rgb(176, 18, 235);
font: 14pt &quot;Magnum&quot;;
border-radius: 15px;
border: 2px solid rgb(176, 18, 235);</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
           <widget class="Line" name="line_9">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>431</width>
              <height>301</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 32);
</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>411</width>
              <height>271</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
             <item>
              <widget class="QSlider" name="slider_time">
               <property name="styleSheet">
                <string notr="true">QSlider::groove:horizontal {
    border: 1px solid rgb(176, 18, 235);
    height: 8px;
    background: rgb(176, 18, 235);
}

QSlider::handle:horizontal {
    background: rgb(176, 18, 235);

    margin: -12px;
}

QSlider::add-page:horizontal {
    background-color: black;
    border: 2px solid rgb(176, 18, 235);
}</string>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="indicator_position">
                   <property name="styleSheet">
                    <string notr="true">color: rgb(176, 18, 253);
font: 14pt &quot;Magnum&quot;;</string>
                   </property>
                   <property name="text">
                    <string>00:00</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>108</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="indicator_time">
                   <property name="styleSheet">
                    <string notr="true">color: rgb(176, 18, 253);
font: 14pt &quot;Magnum&quot;;</string>
                   </property>
                   <property name="text">
                    <string>00:00</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
                 <item>
                  <widget class="QPushButton" name="bt_prev">
                   <property name="font">
                    <font>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>Graphics/перемотка 2.png</normaloff>Graphics/перемотка 2.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bt_pause">
                   <property name="font">
                    <font>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>Graphics/продолжить.png</normaloff>Graphics/продолжить.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bt_play">
                   <property name="font">
                    <font>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>Graphics/пауза.png</normaloff>Graphics/пауза.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bt_next">
                   <property name="font">
                    <font>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>Graphics/перемотка.png</normaloff>Graphics/перемотка.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>25</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="bt_volume_0">
                   <property name="font">
                    <font>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>Graphics/без звука.png</normaloff>Graphics/без звука.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bt_volume_1">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>Graphics/звук 1.png</normaloff>Graphics/звук 1.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bt_volume_2">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>Graphics/звук 2.png</normaloff>Graphics/звук 2.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="slider_volume">
                   <property name="styleSheet">
                    <string notr="true">QSlider::groove:horizontal {
    border: 1px solid rgb(176, 18, 235);
    height: 4px;
    background: rgb(176, 18, 235);
}

QSlider::handle:horizontal {
    background: rgb(176, 18, 235);
    width: 20px;
    height: 20px;

    left: 8px;
    right: 8px;

    margin: -12px;
    border-radius: 14px;
}

QSlider::add-page:horizontal {
    background-color: white;
    border: 1x solid white;
}</string>
                   </property>
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>50</number>
                   </property>
                   <property name="sliderPosition">
                    <number>50</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="indicator_volume">
                   <property name="styleSheet">
                    <string notr="true">color: rgb(176, 18, 253);
font: 14pt &quot;Magnum&quot;;</string>
                   </property>
                   <property name="text">
                    <string>50</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>411</width>
            <height>461</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">color: rgb(176, 18, 235);
font: 14pt &quot;Magnum&quot;;
border-radius: 15px;
border: 2px solid rgb(176, 18, 235);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Список доступной музыки:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="list_music">
             <property name="styleSheet">
              <string notr="true">color: rgb(176, 18, 235);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="bt_load">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>510</y>
            <width>251</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(176, 18, 235);
font: 14pt &quot;Magnum&quot;;
border-radius: 15px;
border: 2px solid rgb(176, 18, 235);</string>
          </property>
          <property name="text">
           <string>Музыка для всех</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>Graphics/музыка.png</normaloff>Graphics/музыка.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
